### PR TITLE
Update contribute/development.md

### PR DESCRIPTION
**Pull request by satotake**
Current [Contribution Page](https://gohugo.io/contribute/development/)  in gohugo.io is a little bit older than [Contribution Guide](https://github.com/gohugoio/hugo/blob/master/CONTRIBUTING.md#submitting-patches) in hugo repository.
This PR will update it.

### Changes

- Update version number of prerequisites (trivial)
- Clone Hugo in a directory outside of `GOPATH`
- Install `mage`
- Install dependencies via `go install`
- Build, install, test and format with `mage`

### CODE REVIEW

File: content/en/contribute/development.md [hunk 1]
```diff
@@ -233,19 +246,40 @@ We have developed a [separate Hugo documentation contribution guide][docscontrib
...
 
 ### Test 
 Sometimes changes on the codebase can cause unintended side effects. Or they don't work as expected. Most functions have their own test cases. You can find them in files ending with `_test.go`.
 
-Make sure the commands `go test ./...` passes, and `go build` completes.
+Make sure the commands 
+
+```
+mage hugoRace
+mage -v check
```
Comment (bep) [inline content/en/contribute/development.md]: I think this is a little "too much". `mage -v check` should be enough.

Comment (satotake) [reply content/en/contribute/development.md]: I see. I deleted `mage -v check`.

File: content/en/contribute/development.md [hunk 2]
```diff
@@ -233,19 +246,39 @@ We have developed a [separate Hugo documentation contribution guide][docscontrib
...
+
+```
+mage -v check
+```
+
+passes, and 
+
+```
+mage hugo
+```
```
Comment (bep) [inline content/en/contribute/development.md]: I meant to say that `mage -v check` should be enough so you can remove this too.

Comment (satotake) [reply content/en/contribute/development.md]: Sorry that I was not able to get what you meant.
Removed.

**Pull request merged**